engine/language-server/src/main/scala/org/enso/languageserver/boot/resource/RepoInitialization.scala

```diff
@@ -26,31 +34,113 @@ class RepoInitialization(
   /** @inheritdoc */
   override def init(): Future[InitializationComponent.Initialized.type] =
     for {
-      _ <- Future.sequence(Seq(suggestionsRepoInit, versionsRepoInit))
+      _ <- suggestionsRepoInit
+      _ <- versionsRepoInit
     } yield InitializationComponent.Initialized
 
   private def suggestionsRepoInit: Future[Unit] = {
-    val initAction = suggestionsRepo.init
+    val initAction =
+      for {
+        _ <- Future {
+          log.info("Initializing suggestions repo.")
```

Comment:
Where is it being initialized? You need to log actually useful information!

Reply:
The source class is included in the log entry.
```
[info] [2021-04-28T07:41:24.949Z] [org.enso.languageserver.boot.resource.RepoInitialization] Initializing suggestions repo.
```

Reply:
Included the database file being initialized :+1: